alf/environments/suite_carla.py

```diff
@@ -1320,7 +1320,9 @@ def _spawn_walkers(self):
                 continue
             walker = self._world.get_actor(response.actor_id)
             self._walkers.append({"walker": walker})
+            self._alf_world.add_actor(walker)
```

Comment:
It seems we have already added the walker in the existing code?

https://github.com/HorizonRobotics/alf/blob/24ab970d61307c66027a7b140ea0606a65b32ee2/alf/environments/suite_carla.py#L1357

Reply:
good catch!